SensorMap.cpp

```diff
@@ -113,15 +113,15 @@ float SensorMap::temperatureCelcius(const int mlx) {
 }
 
 bool SensorMap::blackTile() {
-  i2c_->tcaselect(3);
+  i2c_->tcaselect(kColoSensorID);
   uint16_t r, g, b, c;
   tcs_.getRawData(&r, &g, &b, &c);
   return (r < 115 && g < 115 && b < 115);
 }
 
 bool SensorMap::silverTile() {  
-  i2c_->tcaselect(3);
+  i2c_->tcaselect(kColoSensorID);
   uint16_t r, g, b, c;
   tcs_.getRawData(&r, &g, &b, &c);
-  return (r < 254 && g < 254 && b < 254);
+  return (r > 254 && g > 254 && b > 254);
```

Comment:
It would be convenient to add a comment saying the common values you see when detecting the silver tile. Also, makes this values constants but you can do that on a later PR. Please add a TODO regarding this.

Reply:
not what I meant, reason why having this is wrong:
const uint16_t r, g, b, c;
tcs_.getRawData(&r, &g, &b, &c);

I meant making const later the 254 and 115, but for the moment simply leave the TODO.

Reply:
Perfect, let me change it and upload it